### PR TITLE
@eessex => Markdown supported messaging

### PR DESCRIPTION
**Pull request by kanaabe**
Testing danger too.....

image

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by kanaabe** [comment]
YESSSS FINALLY!

File: client/apps/edit/components/admin/super_article/index.styl [hunk 1]
```diff
@@ -23,4 +23,8 @@ section.image-upload-form.disabled
   .image-upload-form, textarea
     height 175px
   .image-upload-form h1
-    margin-top 60px
\ No newline at end of file
+    margin-top 60px
+  &__markdown
```
Comment (eessex) [inline client/apps/edit/components/admin/super_article/index.styl]: should we add this to the generic form styles?

Comment (kanaabe) [inline client/apps/edit/components/admin/super_article/index.styl]: suuure i can add that

**Pull request merged**